### ISSUE TITLE
Mesour\Components\Application\Request::getAllHeaders() typo

### ISSUE THREAD

**Issue by oddz** [mesour/components#3]
There is a typo in that function. Headers should be initialized as an array not a string. In php 7.1 with my configuration this error is resulting in a fatal exception in my view.

**Comment by oddz** [comment]
I suspect this is correct.

```
	protected function getAllHeaders()
	{
		if (!function_exists('getallheaders')) {
			$headers = [];
			foreach ($_SERVER as $name => $value) {
				if (substr($name, 0, 5) == 'HTTP_') {
					$headers[str_replace(' ', '-', ucwords(strtolower(str_replace('_', ' ', substr($name, 5)))))] = $value;
				}
			}
			return $headers;
		}
		return getallheaders();
	}
```

**Comment by mesour** [comment]
Sorry i missed this issue, fixed in v3.0.6
